plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SpotlessTask.java

```diff
@@ -168,11 +173,23 @@ public void performAction(IncrementalTaskInputs inputs) throws Exception {
 				.steps(steps)
 				.exceptionPolicy(exceptionPolicy)
 				.build()) {
+			// determine if a list of files has been passed in
+			Predicate<File> shouldInclude;
+			Project project = getProject();
+			if (project.hasProperty(FILES_PROPERTY) && project.property(FILES_PROPERTY) instanceof String) {
+				Object rawIncludePatterns = project.property(FILES_PROPERTY);
+				assert rawIncludePatterns != null;
```

Comment:
Why assert? (The `instanceof` already ensures it's not null, otherwise why not check again it's still a `String`?)

Reply:
i think it was an IntelliJ suggestion that I applied without thinking. Fixed